l10n_co_geo/data/res_country_state.xml

```diff
@@ -0,0 +1,204 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I just saw that those get increasingly merged into `upstream/10.0`. That would be the most elegant of all solutions, i guess...